connect/transforms/src/main/java/org/apache/kafka/connect/transforms/TimestampConverter.java

```diff
@@ -68,6 +68,9 @@ public abstract class TimestampConverter<R extends ConnectRecord<R>> implements
     public static final String UNIX_PRECISION_CONFIG = "unix.precision";
     private static final String UNIX_PRECISION_DEFAULT = "milliseconds";
 
+    public static final String TIMEZONE_CONFIG = "time_zone";
```

Comment:
```suggestion
    public static final String TIMEZONE_CONFIG = "time.zone";
```

Consistent naming please :)